templates/web/fixmystreet.com/footer_extra_js.html

```diff
@@ -21,6 +21,7 @@
     scripts.push( version('/cobrands/tfl/assets.js') );
     scripts.push( version('/cobrands/highways/assets.js') );
     scripts.push( version('/cobrands/fixmystreet-uk-councils/council_validation_rules.js') );
+    scripts.push( version('/cobrands/fixmystreet-uk-councils/roadworks.js') );
```

Comment:
This does mean we'll display this everywhere, which I guess is fine.

Reply:
Oh yes, I probably left that in for testing.